### PR TITLE
Support for minWidth property

### PR DESCRIPTION
**Pull request by burtek**
Tested in own project

Includes changes from `npm audit`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ const MyComponent = () => {
 
 _onAutoSize={(newWidth) => {}}_
 
-Callback function to be fired on input resize.
+Callback function to be fired on input resize. `newWidth` does not include width specified by `extraWidth` (see [above for `extraWidth` prop](#extrawidth))
```
Comment (burtek) [inline README.md:94]: Learned it the hard way, so felt like it should be noted here

File: src/index.tsx [hunk 2]
```diff
@@ -115,21 +116,25 @@ const AutowidthInput: React.FC<AutowidthInputProps> = forwardRef<
...
-                setInputWidth(placeholderWidth + +extraWidth);
+                setInputWidth(Math.max(+minWidth, placeholderWidth) + +extraWidth);
 
                 if (onAutosize) onAutosize(placeholderWidth);
             } else if (sizerRef.current) {
                 /* If no input value or placeholder exists, update the sizer to the width of the "extraWidth" prop (default is 16) */
 
                 setInputWidth(+extraWidth);
-                if (onAutosize) onAutosize(+extraWidth);
+                if (onAutosize) onAutosize(+minWidth);
```
Comment (burtek) [inline src/index.tsx:137]: This was the only place where `extraWidth` was included in `onAutosize` call, it felt odd and was inconsisted.

**Comment by burtek** [comment]
Should I build the lib and include `dist` in this PR, too? 🤔

File: src/index.tsx (hunk 2 above)
Comment (kierien) [reply src/index.tsx:137]: Would it make more sense to add extraWidth to the other onAutosize callback instead of removing it from this one?

**Comment by kierien** [comment]
> Should I build the lib and include `dist` in this PR, too? 🤔

Yes please!

**Comment by burtek** [comment]
@kierien there you go

File: src/index.tsx (hunk 2 above)
Comment (burtek) [reply src/index.tsx:137]: I mean, one way or another. Also, it need to be consistent with `minWidth` as one might collect sizes from multiple inputs and apply same `minWidth={Math.max(...inputSizes)}` to all of them

**Comment by kierien** [comment]
Thanks @burtek

**Pull request merged**